cmd/spiffe-aws-assume-role/cli/cli.go

```diff
@@ -92,3 +97,17 @@ func parse(args []string) (*kong.Context, error) {
 
 	return parser.Parse(args)
 }
+
+func createSession(stsEndpoint string, stsRegion string) *session.Session {
+	config := &aws.Config{}
+
+	if len(stsEndpoint) > 0 {
+		config.Endpoint = aws.String(stsEndpoint)
+	}
+
+	if len(stsRegion) > 0 {
+		config.Region = aws.String(stsRegion)
+	}
+
+	return session.Must(session.NewSession(config))
+}
```

Comment:
Moved from another file, simplified (we poke at the default config rather than spinning up a new instance), with added support for the STS region (this is the functional crux of the change).